src/molecule/provisioner/ansible.py

```diff
@@ -945,7 +946,7 @@ def _get_modules_directories(self):
             ]
         )
 
-        return paths
+        return [path for path in paths if path is not None]
```

Comment:
If I understand this method correctly, the only source of `None`s in the `ANSIBLE_LIBRARY` environment variable (`path/1::path/2` will result in `["/abs/path/1", None, "/abs/path/2"]`). So maybe we should eliminate bad values there?

Reply:
That is exactly what we do here, assure we do not return None in that list.

Reply:
But we are doing it in the wrong place. This should be done in line 920 where `None`s can be introduced.

Reply:
By looking at the body of the function I feel far more confident to remove None values at the end. There are few other areas which could return None, especially as `util.abs_path` can return None.

I do not remember exactly the error that motivated me to make this PR, but my impression is that the None came from `_get_plugin_directory` and not from `ANSIBLE_LIBRARY`.

As that used to raise an ugly runtime exception, I prefer to sanitize the values before returning the list.